src/web_app/js/util.js

```diff
@@ -89,17 +85,12 @@ function requestTurnServers(turnRequestUrl, turnTransports) {
       }
       // Filter the TURN URLs to only use the desired transport, if specified.
       if (turnTransports.length > 0) {
-        filterTurnUrls(turnServerResponse.uris, turnTransports);
+        for (var i = 0; i < turnServerResponse.iceServers.length; i++) {
+          filterTurnUrls(turnServerResponse.iceServers[i].urls, turnTransports);
```

Comment:
Remember to drop the ice servers that end up wuth an empty urls. Maybe copy the filterIceConfig method from testrtc.